Formula/e/easyrpg-player.rb

```diff
@@ -35,6 +35,13 @@ class EasyrpgPlayer < Formula
   depends_on "sdl2"
   depends_on "speexdsp"
 
+  uses_from_macos "zlib"
+
+  on_macos do
+    depends_on "expat"
```

Comment:
```ruby
    uses_from_macos "expat"
```

Reply:
👍

Reply:
why do we need linkage declaration for linux?